### ISSUE TITLE
optical flow training

### ISSUE THREAD

**Issue by zhangwlgq** [hzwer/arXiv2020-RIFE#100]
Hi, I am trying to train the IFNet from scratch on my own dataset. But loss could not converge and output flow value does not even exceed 1.0 . The data format should be correct because the warp function works fine. The loss function is L1 loss between predicted flow and gt flow. 

Is there any trick or point need to be noted when training the IFNet? Thank you.

**Comment by hzwer** [comment]
看您主页我推测您是国人，因此用中文回答。这个问题挺难回答的，按理说不存在什么 trick，推测是训练代码有 bug。比如说你是否合理设置了 batchsize（不能是1） 和 学习率（一般不超过1e-3），当你固定训练一个 batch 时 loss 会下降吗？

**Comment by zhangwlgq** [comment]
谢谢。我试过多种参数组合，也试过在单一数据下做过拟合，结果都不好。我也认为是代码某处存在bug，只不过之前没有训练flownet的经验，怀疑有什么点忽视了，我再仔细检查一下。

**Comment by hzwer** [comment]
新的模型不需要光流标签了，因此关闭该问题
